pyproject.toml

```diff
@@ -1,2 +1,30 @@
 [build-system]
-requires = ["setuptools", "wheel", "katversion"]
+requires = ["setuptools", "katversion"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "switch_exporter"
+dynamic = ["version"]
+authors = [{name = "MeerKAT SDP team", email = "sdpdev+switch_exporter@ska.ac.za"}]
```

Comment:
You might want to update this email alias too.

Reply:
Dang, thought I had. Fixed now.